Makefile

```diff
@@ -18,6 +18,8 @@ NOTIFICATION_SERVICE_IMAGE=notification-service
 
 # get version from pom
 VERSION = $(shell sed -n "s/<revision>\(.*\)<\/revision>/\1/p" $(PWD)/pom.xml | sed 's/\ *//g')
```

Comment:
I think it's safer to use something like 'mvn help:evaluate -Dexpression=project.version -q -DforceStdout' to get the project version. but if we can't use mvn cmd here then this is fine

Reply:
I will explore

Reply:
I could use mvn here but we would not be able to run `make mvn-checks` if maven not installed. Is there any other non maven command we could use here?

Reply:
I don't think so. let's keep sed then